INSTALL.md

```diff
@@ -1,10 +1,4 @@
-# Third Party Repositories
-
-For distributions that do not ship Alacritty officially, there are some third
-party packages available. Only the cargo package is maintained officially, so
-proceed with caution.
-
-## Cargo
+# Cargo Insallation
```

Comment:
Removed mention of Third Party Repositories and changed Cargo section in Cargo Installation section.

Reply:
Minor typo: Cargo Ins_t_allation.